src/Ting/Exceptions/ValueException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace CCMBenchmark\Ting\Exceptions;
+
+use CCMBenchmark\Ting\Exception;
+
+class ValueException extends Exception
```

Comment:
Maybe ConfigException ? SetupException ?

Reply:
Hmmm, I meant this one to be a little more generic, especially for the _Undefined property %s in your criteria_.
I think we might go for yet another new exception type for the config error

Reply:
I misreaded the code, i think ValueException should stay as this.